scripts/build/companion_tools/050-make.sh

```diff
@@ -23,7 +23,7 @@ do_companion_tools_make_build() {
     CT_DoExecLog ALL ${make}
     CT_DoExecLog ALL ${make} install
     if [ "${CT_COMP_TOOLS_make_gmake}" = "y" ]; then
-        CT_DoExecLog ALL ln -sv ${make} "${CT_BUILDTOOLS_PREFIX_DIR}/bin/gmake"
+        CT_DoExecLog ALL ln -sv make "${CT_BUILDTOOLS_PREFIX_DIR}/bin/gmake"
```

Comment:
That's the right change, but it is only half of the fix: the ct-ng will still invoke /usr/bin/make down the road. It looks like https://github.com/crosstool-ng/crosstool-ng/pull/268/commits/6f8e89cb5ca061e899bf3feaaf3fecf30d366c3e by @bhundven went a bit too far: it replaced all "foo" invocations with "${foo}" for make/sed/awk, etc. This is right for the tools that we don't build and rely on the configure to detect the path (sed/awk/...); but for make, we should always be invoking the one in `.build/buildtools/bin`.